controllers/bootstrap/init.go

```diff
@@ -432,3 +480,35 @@ func (b *Bootstrap) deleteSubscription(name, namespace string) error {
 
 	return nil
 }
+
+// Get ConfigMap of Common Services Maps
+func (b *Bootstrap) getCmOfMapCs() (*corev1.ConfigMap, error) {
+	cmName := constant.CsMapConfigMap
+	cmNs := "kube-public"
+	cm := &corev1.ConfigMap{}
+	err := b.Reader.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: cmNs}, cm)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			klog.Errorf("NotFound ConfigMap: %s %s", cmNs, cmName)
+			return nil, nil
+		}
+		klog.Errorf("Failed to get ConfigMap %s %s", cmNs, cmName)
+		return nil, err
+	}
+	return cm, nil
+}
+
+// GetMasterNs gets MasterNamespaces of deploying Common Services
+func (b *Bootstrap) GetMasterNs() (string, bool) {
+	operatorNs, err := util.GetOperatorNamespace()
+	if err != nil {
+		klog.Errorf("Getting operator namespace failed: %v", err)
+		return "", false
+	}
+
+	cm, _ := b.getCmOfMapCs()
+	if masterNs, found := cm.Data[operatorNs]; found {
+		return masterNs, true
```

Comment:
If the master namespace isn't found in the configmap, we should set the MasterNamespace to `ibm-common-services`